src/applications/mhv/secure-messaging/components/MessageThread/MessageThread.jsx

```diff
@@ -7,7 +7,7 @@ import { clearMessageHistory } from '../../actions/messages';
 
 const MessageThread = props => {
   const dispatch = useDispatch();
-  const message = useSelector(state => state.sm.messageDetails.message);
+  // const message = useSelector(state => state.sm.messageDetails.message);

```

Comment:
Do you still need this line? If not, may consider deleting it.

Reply:
@vsaleem great catch, I meant to remove it.